app/services/foreman/wrapped_exception.rb

```diff
@@ -13,7 +13,7 @@ def message
       super unless @wrapped_exception.present?
 
       cls = @wrapped_exception.class.name
-      msg = @wrapped_exception.message.try(:truncate, 90)
+      msg = @wrapped_exception.message
       super + " ([#{cls}]: #{msg})"
     end
   end
```

Comment:
There is one more thing - `WrappedException` class is missing `backtrace` method, therefore if sent to `Foreman::Logging.exception` it fails with `no implicit conversion of nil into Array`. Since this is an enhancement PR, could you add this method and a test here?